sdks/wasm/ProxyDriver/Program.cs

```diff
@@ -55,6 +55,10 @@ public static Task Start (string chromePath, string appPath, string pagePath)
 
 				host = new WebHostBuilder()
 					.UseSetting ("UseIISIntegration", false.ToString ())
+					.ConfigureAppConfiguration ((hostingContext, config) => {
+						config.AddJsonFile ("appsettings.json", optional: true, reloadOnChange: true);
+						config.AddEnvironmentVariables (prefix: "WASM_TESTS_");
+					})
 					.ConfigureServices (services => {
 						services.Configure<TestHarnessOptions> (options => {
 							options.ChromePath = chromePath;
```

Comment:
I think you can probably avoid all the TestHarnessStartup changes with:
```suggestion
							options.ChromePath = options.ChromePath ?? chromePath;
```